src/main/java/net/sf/jabref/gui/specialfields/SpecialFieldIcon.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.jabref.gui.specialfields;
+
+import javax.swing.Icon;
+import javax.swing.JLabel;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.specialfields.SpecialField;
+import net.sf.jabref.model.entry.SpecialFieldValue;
+
+
+public class SpecialFieldIcon {
+
+    public static Icon getRepresentingIcon(SpecialField field) {
+        if (SpecialField.PRINTED.equals(field)) {
```

Comment:
No. That's not possible/desired as the enum `SpecialField` is part of `logic` and should not contain UI elements (such as `JabRefIcon`)

@lenhard Icons for `Priority` and `Read Status` in the Icon bar are still too small :wink: